src/screens/QrScanner.js

```diff
@@ -207,3 +212,45 @@ const styles = StyleSheet.create({
     paddingBottom: 20,
   }
 });
+
+function rawDataToU8A(rawData) {
+  if (!rawData) {
+    return null;
+  }
+
+  // Strip SQRC filler bytes
```

Comment:
```suggestion
  // Strip SQRC filler bytes
  if (rawData.substr(-2) === 'ec') {
    rawData = rawData.substr(0, rawData.length - 2);
  }
```

Reply:
Ah, should have looked here first, did exact same thing myself just now :P.